language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -518,6 +522,19 @@ impl ResolvingGraph {
     fn download_and_update_if_repo(dep_name: PackageName, dep: &Dependency) -> Result<()> {
         if let Some(git_info) = &dep.git_info {
             if !git_info.download_to.exists() {
+                let git_url = git_info.git_url.clone().to_string();
```

Comment:
This is bumping the Movey download count, right?

If so, please add this change in a separate PR. This isn't related to the stated purpose of the PR (*uploading* metadata), and I want to make sure we discuss this separately (including impact on build perf, what to do if Movey crashes, and adding flags for opting out).